lambdas/common/config/config-service.ts

```diff
@@ -51,6 +52,25 @@ export class ConfigService {
     }
   }
 
+  public getAuditConfig(): CriAuditConfig {
```

Comment:
I know this was the original design, but I wonder if we need `CriAuditConfig` the only thing that come from config is the issuer. Perhaps the bulk of this should move into the Audit code and then we just pass in the issuer.